test/appium/views/sign_in_view.py

```diff
@@ -233,18 +246,23 @@ def create_user(self, password=common_password, keycard=False, enable_notificati
         self.driver.info("## New multiaccount is created successfully!", device=False)
         return self.get_home_view()
 
-    def recover_access(self, passphrase: str, password: str = common_password+'1234', keycard=False,
+    def recover_access(self, passphrase: str, password: str = common_password, keycard=False,
                        enable_notifications=False, second_user=False, username='Restore user', set_image=False):
         self.driver.info("## Recover access(password:%s, keycard:%s)" % (password, str(keycard)), device=False)
 
         if not second_user:
             self.i_m_new_in_status_button.click_until_presence_of_element(self.generate_keys_button)
+        else:
+            self.show_profiles_button.click()
+            self.plus_profiles_button.click()
+            self.create_new_profile_button.click()
```

Comment:
Is there any sense to use click_until_presence_of_element() method or it doesn't matter? 

self.create_new_profile_button.click_until_presence_of_element(self.generate_keys_button)

Reply:
well, in case if it would fail. I prefer to use it in places where it is fragile, so all the places where it is used have had at least one failure on the matter